src/verinfast/utils/utils.py

```diff
@@ -147,3 +147,31 @@ def log(self, msg, tag=None, display=False, timestamp=True):
             f.write(output+"\n")
         if display or self.debug:
             print(output)
+
+
+# Returns a tuple of the repo name and the repo url from the original url
+def get_repo_name_url_and_branch(repo_url: str):
+    # match = re.search(r"([^/]*\.git.*)", repo_url) ^.*?/(.*)
```

Comment:
Can we remove this? it's really confusing to have two commented out lines that are basically the same thing.

Reply:
Fixed. I also set the agent to use the new function